Ed: Rewriting of osm2ed

l-vincent-l opened the pull request:
We are now able to compute netherlands with 1,2 GB of RAM

I think we can make some time and RAM improvement by using int32 in polygons

l-vincent-l committed:
Ed: better initialization of PoiType

l-vincent-l committed:
Ed: init poi with OSM data

l-vincent-l committed:
Ed: change pointers to objects

l-vincent-l committed:
Ed: Some lint in osm_tags_reader

l-vincent-l committed:
Ed: complete rewriting of osm2ed

l-vincent-l committed:
Ed: correct cmakes

l-vincent-l committed:
Ed: Make some functions public

l-vincent-l committed:
Ed: add a forgotten file

l-vincent-l committed:
Ed: add gcc4.7.2 compatibility

kinnou02 commented on source/ed/osm2ed.cpp:
```diff
@@ -486,45 +785,29 @@ int main(int argc, char** argv) {
+    ed::EdPersistor persistor(connection_string);
+    persistor.clean_georef();
+    persistor.clean_poi();
+    po::notify(vm);
```
it's already done line 787

kinnou02 committed:
Osm2ed: fix a segfault

we didn't check the result of a find, so it happens that we take a
pointer to the warp

kinnou02 committed:
osm2ed: some lint

kinnou02 commented:
J'ai une branche avec un fix et du lint a rebaser correctement: https://github.com/kinnou02/navitia/tree/bigger_osm

l-vincent-l committed:
OSM2Ed: try to insert less nodes
